### ISSUE TITLE
M5 Consumption Reporting - ERROR: Failed to store Consumption Report for provisioning session 

### ISSUE THREAD

**Issue by dsilhavy** [5G-MAG/rt-5gms-application-function#115]
## Description
I am currently testing the end-to-end chain for consumption reporting . This includes:

* Version 1.1.0 of the UE components (currently in the `development` branch)
* Version 1.4.0 of the AF (currently in the `development` branch)
* Version 1.2.2 of the AS (currently in the `development` branch)

As part of the testing I identified a minor issue in the Media Session Handler as documented here: https://github.com/5G-MAG/rt-5gms-media-session-handler/issues/52

After fixing the issue above the consumption reports are accepted by the AF and I can see a single report being created:

````json
{
    "mediaPlayerEntry": "http://192.168.2.7/m4d/provisioning-session-527136fe-b07f-41ee-b0e8-039343ecf354/bbb/2/client_manifest-common_init.mpd",
    "reportingClientId": "5edfc1ad-40ac-4b1e-8380-bc5fd466e223",
    "consumptionReportingUnits":
    [
        {
            "mediaConsumed": "1920x1080p25",
            "mediaEndpointAddress":
            {
                "ipv4Addr": "192.168.2.4",
                "portNumber": 80
            },
            "startTime": "2024-01-11T13:46:37Z",
            "duration": 9,
            "locations":
            []
        },
        {
            "mediaConsumed": "160kbps",
            "mediaEndpointAddress":
            {
                "ipv4Addr": "192.168.2.4",
                "portNumber": 80
            },
            "startTime": "2024-01-11T13:46:37Z",
            "duration": 9,
            "locations":
            []
        }
    ]
}
````

After creating the first consumption report the Application Function is throwing an error for each of the following reports:

````
01/11 12:48:07.400: [msaf] ERROR: Failed to store Consumption Report for provisioning session [527136fe-b07f-41ee-b0e8-039343ecf354] (../src/5gmsaf/msaf-m5-sm.c:793)
01/11 12:48:18.264: [msaf] ERROR: Unable to create /var/local/log/open5gs/reports/527136fe-b07f-41ee-b0e8-039343ecf354/consumption_reports/5edfc1ad-40ac-4b1e-8380-bc5fd466e223_2024-01-11T13:46:37Z.json for writing: File exists (../src/5gmsaf/data-collection.c:111)
01/11 12:48:18.264: [msaf] ERROR: Failed to store Consumption Report for provisioning session [527136fe-b07f-41ee-b0e8-039343ecf354] (../src/5gmsaf/msaf-m5-sm.c:793)
01/11 12:48:27.492: [msaf] ERROR: Unable to create /var/local/log/open5gs/reports/527136fe-b07f-41ee-b0e8-039343ecf354/consumption_reports/5edfc1ad-40ac-4b1e-8380-bc5fd466e223_2024-01-11T13:46:37Z.json for writing: File exists (../src/5gmsaf/data-
````

It looks like for some reason the time in the filename is not increased and the existing file is not overwritten. The response code of the Application Function is 500.

**Comment by dsilhavy** [comment]
Discussion during the 5G-MAG call: 
* Each consumption report should be saved in a new file
* The filename should include the reception time of the report

**Comment by davidjwbbc** [comment]
Closed by PR #118
